### PR TITLE
Mock adapter

### PR DESCRIPTION
**Pull request by itsthatguy**
- Use the mock adapter to develop in the browser
- Starts a pattern for multiple adapters

### CODE REVIEW

File: src/app/messages/messages.input.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import MessagesActions from '../messages/messages.actions';
```
Comment (TylerK) [inline src/app/messages/messages.input.jsx]: Is this file duplicating the app/message_input/message_input.jsx?

File: src/app/messages/messages.input.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    if (event.keyCode === 13) {
+      MessagesActions.sendMessage(event.target.value);
+      this.setState({value: ''});
+    }
+  },
+
+  render () {
+    return (
+      <div>
+        <input type='text' value={this.state.value} onKeyDown={this.onKeyDown} />
```
Comment (TylerK) [inline src/app/messages/messages.input.jsx]: Always want onChange to account for pastes and other actions

Comment (itsthatguy) [inline src/app/messages/messages.input.jsx]: replace this with your work... this was planned to be temporary

File: src/app/messages/messages.input.jsx (hunk 1 above)
Comment (itsthatguy) [inline src/app/messages/messages.input.jsx]: no

Comment (itsthatguy) [inline src/app/messages/messages.input.jsx]: remove this, add yours

**Comment by itsthatguy** [comment]
This should be good to go now.

**Pull request merged**